docs/guides/nwaku/run-docker.md

```diff
@@ -44,7 +38,7 @@ docker run [OPTIONS] [IMAGE] [ARG...]
 Run `nwaku` using the most typical configuration:
 
 ```bash
-docker run -i -t -p 60000:60000 -p 9000:9000/udp statusteam/nim-waku \
+docker run -i -t -p 60000:60000 -p 9000:9000/udp statusteam/nim-waku:v0.18.0 \
```

Comment:
Notice that this version has the `nim-libp2p` bug where the node stopped accepting new incoming `js-waku` connections. I think the fix is available in `v0.19.0-rc.0` although not sure. I'm sure the fix is in `'ivansete/nim-waku:wakunode2-v0.18.0-30-g84db7b'`.

That makes me think that is interesting to had the commit hash within the rc tag name

Reply:
You can go to https://github.com/waku-org/nwaku/releases/tag/v0.19.0-rc.0 and see all the commits that are in that release:) That is why releases use `tags`, so that it is easy to figure out which commits are in:) 

REgarding the bug, we can point to `v0.19.0-rc.0`, or just keep this PR from merging until next week when we release 0.19.0. No strong preference

Reply:
I was about to come and mention "dismiss my comment". The `v0.19.0-rc.0` is itself a great reference!

Re the bug, better reference the `v0.19.0-rc.0` and change it next week.